model/registry/vcs.yaml

```diff
@@ -83,3 +94,18 @@ groups:
           The ID of the change (pull request/merge request) if applicable. This
           is usually a unique (within repository) identifier generated by the VCS system.
         examples: ["123"]
+      - id: vcs.change.state
+        type:
+          members:
+            - id: open
+              value: open
+              brief: "[open](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefbranchabranch)"
+              stability: experimental
+            - id: merged
+              value: merged
+              brief: "[merged](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddeftagatag)"
+              stability: experimental
+        stability: experimental
+        brief: >
+          The state of a change (pull request).
```

Comment:
```suggestion
          The state of the change (pull request/merge request).
```

Reply:
Done in 347b15b0db0a0451f0ae696d949b7d4537819e87